### PR TITLE
Added hardhat dependency missing commad and Solution to  NPM Error on windows

### PR DESCRIPTION
**Pull request by 0xumarkhatab**
Kindly view the commit messages

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,10 +135,17 @@ At that point, utilizing smart contract storage directly is just infeasible and
 ### Prerequisites
 - Please learn the basics of Mocha and Chai if you are not aware of them, to understand what they are follow this [tutorial](https://medium.com/spidernitt/testing-with-mocha-and-chai-b8da8d2e10f2)
 
+###### Note
```
Comment (haardikk21) [inline README.md]: Prefix this with a `>` instead of the `###### Note`

```
> Note: ...
```

File: README.md [hunk 2]
```diff
@@ -135,10 +135,17 @@ At that point, utilizing smart contract storage directly is just infeasible and
...
+ All of these commands should work smoothly . 
+ If you are on windows and face Errors Like `Cannot read properties of null (reading 'pickAlgorithm')`
+ Try Clearing the NPM cache using `npm cache clear --force`
+ 
 ----
 
 Let's see how all this works practically for our whitelist example.
 
+        
+
```
Comment (haardikk21) [inline README.md]: remove extra newline

File: README.md [hunk 3]
```diff
@@ -149,7 +156,7 @@ Let's see how all this works practically for our whitelist example.
 - If you are on a Windows machine, please do this extra step and install these libraries as well :)
 
   ```bash
-  npm install --save-dev @nomiclabs/hardhat-waffle ethereum-waffle chai @nomiclabs/hardhat-ethers ethers
+  npm install --save-dev @nomicfoundation/hardhat-toolbox @nomiclabs/hardhat-waffle ethereum-waffle chai @nomiclabs/hardhat-ethers ethers
```
Comment (haardikk21) [inline README.md:157]: Thanks

**Review by haardikk21 — CHANGES_REQUESTED** [review]

**Comment by 0xumarkhatab** [comment]
Changes are Incorporated

**Review by haardikk21 — APPROVED** [review]
Banging

**Pull request merged**